testsuite/test-generate.sh

```diff
@@ -606,6 +607,14 @@ if [ "${tensorflowtests}" == "y" ]; then
         GenerateTensorflowStandardTests
     fi
 fi
+if [ "${tensorflowhorovodtests}" == "y" ]; then
+    if [ "${standardtests}" == "y" ]; then
+        if [[ ${submissiontype} == *mpirun ]]
```

Comment:
i don't think you need to check the submission type?  The `if [ -a ...` check in the test generation should be enough I think?

Reply:
You are right :) Removing it.